store/postgres/src/subgraphs.graphql

```diff
@@ -71,7 +71,7 @@ type DynamicEthereumContractDataSource @entity {
 
 type EthereumContractSource @entity {
     id: ID!
-    address: String!
+    address: Bytes
```

Comment:
Contracts always have an address, I'm pretty sure, so we should keep the field as required with `!`.

Reply:
Unfortunately, they don't; we have 29 `EthereumContractSource` in hosted without an address. They belong to these subgraphs: 
```
ensdomains/ens
ensdomains/ensgoerli
ensdomains/ensrinkeby
ensdomains/ensropsten
luckyyang/ela-ens
luckyyang/elastos-eth-ens
makoto/ens-goerli
makoto/ens-mainnet
makoto/ens-subgraph-goerli-text
makoto/ens-subgraph-goerli-text-indexed
radek1st/metamultisig
radek1st/meta-multisig
radek1st/multisig
wighawag/eip721-subgraph
```

The `EthereumContractSource` in question are
```
QmPcFGJPhiq7NXUXeBUUiVbjv5D1RKKsg5Y29GNmP838bf-manifest-data-source-5-source
QmReUGTmnBqrjZierx4rvjgRMDxkZ5icHa6ECv1WfdUvw9-manifest-data-source-5-source
QmU7igtqrBQhLeD6SafF6YqUMvrtkbgfJCPmM17NRbneyR-manifest-data-source-1-source
QmTBmMKqcDH7GQegG6ktk6hWFzi2YRXAJU3XSvBrXPovJJ-manifest-data-source-1-source
QmRYNumG74pNtCA4jGftJkJm3hBgapEvDFN6yojjdBypm4-manifest-data-source-0-source
QmVEoWSQ8eNnVkL1uw2toDSc1xXKi1ADYUm36p961Q1EzT-manifest-data-source-5-source
QmZTHHbtEyGcDRp2mWiuSHMKU1PrHKh7KaMp1jDPSNDqRv-manifest-data-source-1-source
QmafxPzuuWFtLzxkEXMVBFpkkKcwavaK2D9BSLFDyMHQxK-manifest-data-source-1-source
QmXieLMydGH6rT8bN1jCi5JeLmsfkyjNKhxGoveWffVxBe-manifest-data-source-5-source
QmZUg1pXA7izP1TQ6krMy7wS2YzHSTbCwjnUPtfJnL2RkN-manifest-data-source-1-source
QmeqpV7maxD3pFZmcVSLyAuDyhFL6E7Vfw39ZmG1P1cdMM-manifest-data-source-0-source
QmTBmMKqcDH7GQegG6ktk6hWFzi2YRXAJU3XSvBrXPovJJ-manifest-data-source-5-source
QmUHceFFmgeR4tBF3F38DsZiGvozx6CEM4H5xXw7JYeU4t-manifest-data-source-1-source
QmReUGTmnBqrjZierx4rvjgRMDxkZ5icHa6ECv1WfdUvw9-manifest-data-source-1-source
QmQ491oCHnZiRS9LtS9nsTfbPd6EGLpcebcwMzUsou5aM7-manifest-data-source-1-source
QmU7igtqrBQhLeD6SafF6YqUMvrtkbgfJCPmM17NRbneyR-manifest-data-source-5-source
QmVEoWSQ8eNnVkL1uw2toDSc1xXKi1ADYUm36p961Q1EzT-manifest-data-source-1-source
```

Reply:
Of course, you're right. Addresses are optional on data sources.